### PR TITLE
New version: M2Team.NanaZip version 3.0.1000.0

### LINKED ISSUE #157963: [Update Request]: NanaZip
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

M2Team.NanaZip

### Package Version

2.0.450.0

### Please describe the changes you would like to see

New version available, current package is old, new version: [(3.0.1000.0)](https://github.com/M2Team/NanaZip/releases/tag/3.0.1000.0)

### PR DESCRIPTION
**Pull request by aisgbnok**
Checklist for Pull Requests
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Is there a linked Issue?
  - Resolves #157963

Manifests
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/161889)

### CODE REVIEW

**Comment by aisgbnok** [comment]
It installs fine for me:
```
2024-07-07 23:49:56.114 [CORE] WinGet, version [1.9.1792-preview], activity [{0C23F350-6213-4464-B28A-4F902F16E3DB}]
2024-07-07 23:49:56.115 [CORE] OS: Windows.Desktop v10.0.22635.3858
2024-07-07 23:49:56.115 [CORE] Command line Args: "C:\Users\Anthony\AppData\Local\Microsoft\WindowsApps\winget.exe" install --manifest E:\winget-pkgs\manifests\m\M2Team\NanaZip\3.0.1000.0\
2024-07-07 23:49:56.115 [CORE] Package: Microsoft.DesktopAppInstaller v1.24.1792.0
2024-07-07 23:49:56.115 [CORE] IsCOMCall:0; Caller: winget-cli
2024-07-07 23:49:56.125 [CLI ] WinGet invoked with arguments: 'install' '--manifest' 'E:\winget-pkgs\manifests\m\M2Team\NanaZip\3.0.1000.0\'
2024-07-07 23:49:56.125 [CLI ] Found subcommand: install
2024-07-07 23:49:56.125 [CLI ] Leaf command to execute: root:install
2024-07-07 23:49:56.129 [CLI ] Executing command: install
2024-07-07 23:49:56.131 [CLI ] Manifest fields: Name [NanaZip], Version [3.0.1000.0]
2024-07-07 23:49:56.166 [CORE] Default proxy is not set
2024-07-07 23:49:56.894 [CLI ] Installer hash verified
2024-07-07 23:49:57.194 [CORE] Starting StagePackageAsync operation #0: https://github.com/M2Team/NanaZip/releases/download/3.0.1000.0/NanaZip_3.0.1000.0.msixbundle Options:  { SkipReputationCheck = 0, ExpectedDigests = { { URI = https://github.com/M2Team/NanaZip/releases/download/3.0.1000.0/NanaZip_3.0.1000.0.msixbundle, Digest = AzVK3PpRyXmjygXKVcr08Tdv5ZwNXdb+ie0DNy28Nug= } } }
2024-07-07 23:49:57.250 [CORE] Begin waiting for operation #0
2024-07-07 23:49:57.250 [CORE] Begin blocking for operation #0
2024-07-07 23:50:15.915 [CORE] Successfully completed #0
2024-07-07 23:50:15.915 [CORE] Starting RegisterPackageByFullNameAsync operation #1: 40174MouriNaruto.NanaZip_3.0.1000.0_neutral_~_gnj4mf6z9tkrc
2024-07-07 23:50:15.926 [CORE] Begin waiting for operation #1
2024-07-07 23:50:15.926 [CORE] Begin blocking for operation #1
2024-07-07 23:50:16.486 [CORE] Successfully completed #1
2024-07-07 23:50:16.495 [CLI ] Leaf command succeeded: root:install
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -2147009295
> 2024-07-09 03:40:28.261 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(161)\WindowsPackageManager.dll!5609E94A: (caller: 560097A7) Exception(1) tid(808) 80073CF1 Package was not found.
> 2024-07-09 03:40:28.354 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!5609D7AF: (caller: 5609ED72) Exception(2) tid(808) 80073CF1 Package was not found.
> 2024-07-09 03:40:28.354 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(126)\WindowsPackageManager.dll!5609EA29: (caller: 5609FBF8) LogHr(1) tid(808) 80073CF1 Package was not found.
>     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!5609D7AF: (caller: 5609ED72) Exception(2) tid(808) 80073CF1 Package was not found.
> 2024-07-09 03:40:28.363 [CLI ] MSIX installer failed: 2147958001

(Automated response - build 886.)

File: manifests/m/M2Team/NanaZip/3.0.1000.0/M2Team.NanaZip.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- zip
+- zst
+- zstd
+PackageFamilyName: 40174MouriNaruto.NanaZip_gnj4mf6z9tkrc
+ReleaseDate: 2024-05-27
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/M2Team/NanaZip/releases/download/3.0.1000.0/NanaZip_3.0.1000.0.msixbundle
+  InstallerSha256: 27D4EAD5F9AF51E573145CE1CCA78F063606AB8AE8F7124801D4D887453F5397
+  SignatureSha256: 632B5C4F6A758BED7ABE698217E14294D4D00DF848FAD9070EC6D2F26E50C2DA
```
Comment (stephengillie) [inline manifests/m/M2Team/NanaZip/3.0.1000.0/M2Team.NanaZip.installer.yaml:74]:
```suggestion
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
I also found this, which might be relevant:
> 2024-07-09 03:40:28.260 [CORE] Deployment operation #0: Unable to install bundle https://github.com/M2Team/NanaZip/releases/download/3.0.1000.0/NanaZip_3.0.1000.0.msixbundle because it does not have an appropriate application package for x86 architecture.

**Comment by aisgbnok** [comment]
> I also found this, which might be relevant:
> 
> > 2024-07-09 03:40:28.260 [CORE] Deployment operation #0: Unable to install bundle https://github.com/M2Team/NanaZip/releases/download/3.0.1000.0/NanaZip_3.0.1000.0.msixbundle because it does not have an appropriate application package for x86 architecture.

Yes, thank you for that info. That was the issue. I found this to confirm:

> The 32-bit x86 support for NanaZip packaged version is removed because supported 32-bit x86 Windows versions don't support running on 32-bit only x86 processors.

From [NanaZip 3.0 (3.0.996.0) Release Notes](https://github.com/M2Team/NanaZip/releases/tag/3.0.996#:~:text=The%2032%2Dbit%20x86%20support%20for%20NanaZip%20packaged%20version%20is%20removed%20because%20supported%2032%2Dbit%20x86%20Windows%20versions%20don%27t%20support%20running%20on%2032%2Dbit%20only%20x86%20processors.)

**Review by stephengillie — APPROVED** [review]

**Comment by aisgbnok** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854

**Pull request merged**